### PR TITLE
Add support for web socket sub protocol graphql-transport-ws

### PR DESCRIPTION
**Pull request by toini**

### CODE REVIEW

File: drivers/watty/device.ts [hunk 1]
```diff
@@ -116,8 +116,11 @@ class WattyDevice extends Device {
     }
 
     this.log('Subscribing to live data for homeId', this.#deviceId);
-    this.#wsSubscription = this.#tibber.subscribeToLive(
-      this.subscribeCallback.bind(this),
+
+    this.#wsSubscription = this.#tibber.subscribeToLive().subscribe(
+      (result) => this.subscribeCallback(result),
+      (error) => this.log('Subscription error occurred', error),
+      () => this.log('Subscription ended with no error'),
```
Comment (jstaro) [inline drivers/watty/device.ts:123]: Using observable, nice. Sending in that callback always rubbed me the wrong way.

File: lib/tibber.ts [hunk 2]
```diff
@@ -129,6 +132,8 @@ export class TibberApi {
     this.#homeySettings = homeySettings;
     this.#token = token;
     this.#homeId = homeId;
+    const attributes = getGlobalAttributes();
+    this.#userAgent = `Homey/${attributes.firmwareVersion} com.tibber/${attributes.appVersion}`;
```
Comment (jstaro) [inline lib/tibber.ts:136]: Super minor, but consider destructuring this:
```suggestion
    const { firmwareVersion, appVersion } = getGlobalAttributes();
    this.#userAgent = `Homey/${firmwareVersion} com.tibber/${appVersion}`;
```
for a little bit terser code

**Review by jstaro — APPROVED** [review]
LGTM 👍 
Fewer changes than I expected, actually.

File: lib/tibber.ts (hunk 2 above)
Comment (toini) [reply lib/tibber.ts]: That's miles better :)

**Pull request merged**